pkg/backend/filestate/backend_test.go

```diff
@@ -248,14 +248,14 @@ func TestCancel(t *testing.T) {
 	err = lb.Lock(ctx, aStackRef)
 	assert.NoError(t, err)
 	// check the lock file exists
-	lockExists, err := lb.bucket.Exists(ctx, lb.lockPath(aStackRef.Name()))
```

Comment:
`stackPath` takes a ref, maybe `lockPath` should as well.

Reply:
Changed to take a backend.StackReference since this doesn't currently have need of knowing that it's a `*localBackendReference`.